internal/table/timeseries/timeseries.go

```diff
@@ -96,6 +98,11 @@ func (t *Table) SortBy() string {
 	return t.sortBy
 }
 
+// GetStreams will return the streamers for the table
+func (t *Table) GetStreams() storage.Streamer {
```

Comment:
Instead, let's encapsulate this. This table can instead of having `GetStreams()` implement `Stream(block.Row) error` which makes it a `storage.Streamer`. Then, we can ` stream.Publish(t, s.monitor))`